doc/troubleshooting.md

```diff
@@ -732,6 +759,95 @@ Example format: `remap_rules: "joint_states:=my_joint_states read_single_io:=io/
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
+### Alarm: 8013[12]
+
+*Example:*
+
+```text
+ALARM 8011
+ Empty custom joint name
+[55]
+ALARM 8013
+ group: G, axis: A
+[12]
+```
+
+*Solution:*
+This alarm occurs in conjunction with `8011` (subcode `55`), please refer to [Alarm: 8011[55]](#alarm-801155).
+
+### Alarm: 8013[13]
+
+*Example:*
+
+```text
+ALARM 8013
+ Invalid UserLan port in cfg
+[13]
+```
+
+*Solution:*
+The `userlan_monitor_port` key in the `motoros2_config.yaml` configuration file is set to an invalid value.
+LAN port monitoring will be disabled for this session.
+
+On YRC1000 and YRC1000u, this must be set to either `USER_LAN1` or `USER_LAN2`.
+
+No other values are supported.
+
+Example: `userlan_monitor_port: USER_LAN1`.
+
+After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
+
+### Alarm: 8013[14]
+
+*Example:*
+
+```text
+ALARM 8013
+ UserLan port detect failed
+[14]
+```
+
+*Solution:*
+Because `userlan_monitor_enabled` was set to `true` but no value was supplied for `userlan_monitor_port`, MotoROS2 attempted to auto-detect the LAN port to monitor.
+This auto-detection failed, and as a result MotoROS2 has disabled LAN port monitoring for this session.
+
+To rule out a transient failure, reboot the controller.
+
+If the alarm is raised again, and if auto-detection is not needed or desired, make sure `userlan_monitor_port` is not commented out (ie: does not have a `#` at the start of the line) and set it to an appropriate value.
```

Comment:
> (ie: does not have a `#` at the start of the line)

We may need to emphasize this point at the top of the instructions. I've seen two or three instances of people not removing the `#` on the agent IP address.

Reply:
I don't believe there would be a way to draw attention to that in any other way than repeating the same thing everywhere.

I'd rather add it on a case-by-case basis.